### PR TITLE
Turning into Swift package

### PR DESCRIPTION
**Pull request by jonathanKingston**
**Reviewer:** @GioSensation 
**Asana:** https://app.asana.com/0/0/1202230127172134/f

## Description

This just exports the package out of Autofill as resources that swift can consume. The single swift file exports module do swift can use anything in the dist dir.

### CODE REVIEW

File: .gitattibutes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+dist/** binary
```
Comment (GioSensation) [inline .gitattibutes:1]: Initially I thought "I love this", but then doesn't this make it hard/impossible to commit WIP for the dist files? Sometimes it's useful to commit partial work to unblock other team members. Did you consider this use case. We could just stash everything else and recompile, but I wonder what we're actually gaining by making this a `binary`.

**Review by GioSensation — COMMENTED** [review]
Just a note on marking the dist folder `binary`. What do you think? cc @shakyShane

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            targets: ["Autofill"]),
+    ],
+    dependencies: [
+    ],
+    targets: [
+        .target(
+            name: "Autofill",
+            dependencies: [],
+            resources: [
+                .process("dist")
```
Comment (miasma13) [inline Package.swift:20]: Wouldn't the "copy" instead of "process" rule be more appropriate?

File: .gitattibutes (hunk 1 above)
Comment (jonathanKingston) [reply .gitattibutes:1]: I didn't think of this use-case and personally it kinda sounds like a recipe for disaster; I'm happy to drop this whilst we decide though.

File: Package.swift (hunk 2 above)
Comment (jonathanKingston) [reply Package.swift:20]: I guess I wasn't sure if we might want some of the processing later; but yeah I think for now that's fine. Will change.

Comment (jonathanKingston) [reply Package.swift]: Without .process here it seems we always retain the 'dist' in the file structure which is a little ugly, the alternative is copy each file which is even worse.

I was hoping for .copy('dist/*') or similar syntax that wouldn't keep dist/ in the output.

File: .gitattibutes (hunk 1 above)
Comment (jonathanKingston) [reply .gitattibutes:1]: Spoke with @shakyShane over MM and he thought this is ok, going to mark this as resolved.

File: Package.swift (hunk 2 above)
Comment (jonathanKingston) [reply Package.swift]: Landed on using 'assets' as a dir prefix when speaking with @miasma13 over MM, I think this is the best compromise here.

**Review by GioSensation — APPROVED** [review]
LGTM! 🚀

**Pull request merged**